### ISSUE TITLE
"(736 vs. 100) Dictionary index must be less than base vector's size" thrown in from_utf8

### ISSUE THREAD

**Issue by kagamiori** [facebookincubator/velox#4496]
### Description

Fuzzer caught an error thrown in from_utf8.

### Error Reproduction

Unfortunately, the artifacts persisted via --repro_persist_path didn't reproduce this error for me in velox_expression_runner_test. This error can be reproduced internally by checking out D44643551 and then follow the instruction in the diff summary.

### Relevant logs

```bash
======> Started iteration 1 (seed: 1666535977)
I0403 14:00:23.531760 678908 ExpressionVerifier.cpp:88] Executing expression: and(like(from_utf8("c0","c1"),1oWy]lZR2x~JkQt>0%3cp)vY39X}ZBx|2*3NC;jGGBo(e!,'bP5fKv939>x4{F/%AQ[A;E,qWiCJg(4xK[_[]%H,^!pgm&XsZxssF{58Fa`tz[Nj\7NpQBf(lQ}jZJ<be4UT`8s'R_\{bQIRk$K,S1oE)qG)oShArW]P6stH&9!:|(U!),like(from_utf8("c0","c1"),1oWy]lZR2x~JkQt>0%3cp)vY39X}ZBx|2*3NC;jGGBo(e!,'bP5fKv939>x4{F/%AQ[A;E,qWiCJg(4xK[_[]%H,^!pgm&XsZxssF{58Fa`tz[Nj\7NpQBf(lQ}jZJ<be4UT`8s'R_\{bQIRk$K,S1oE)qG)oShArW]P6stH&9!:|(U!))
I0403 14:00:23.532039 678908 ExpressionVerifier.cpp:31] 2 vectors as input:
I0403 14:00:23.532073 678908 ExpressionVerifier.cpp:33] 	[DICTIONARY VARBINARY: 100 elements, 13 nulls], [DICTIONARY VARBINARY: 100 elements, 11 nulls], [FLAT VARBINARY: 100 elements, 11 nulls]
I0403 14:00:23.532176 678908 ExpressionVerifier.cpp:33] 	[DICTIONARY BIGINT: 100 elements, 10 nulls], [CONSTANT BIGINT: 100 elements, 4844108698146442786]
E0403 14:00:23.533075 678908 Exceptions.h:68] Line: velox/functions/lib/Re2Functions.cpp:952, Function:makeLike, Expression: constantEscape->valueAt(0).size() == 1 (88 vs. 1) Escape string must be a single character, Source: USER, ErrorCode: INVALID_ARGUMENT
I0403 14:00:23.533727 678908 ExpressionVerifier.cpp:141] Modified inputs for common eval path: 
I0403 14:00:23.533775 678908 ExpressionVerifier.cpp:31] 2 vectors as input:
I0403 14:00:23.533829 678908 ExpressionVerifier.cpp:33] 	[DICTIONARY VARBINARY: 100 elements, 13 nulls], [LAZY VARBINARY: 100 elements, no nulls]
I0403 14:00:23.533893 678908 ExpressionVerifier.cpp:33] 	[LAZY BIGINT: 100 elements, no nulls]
E0403 14:00:23.537392 678908 Exceptions.h:68] Line: buck-out/dev/gen/aab7ed39/velox/vector/velox_vector#header-mode-symlink-tree-with-header-map,headers/velox/vector/DictionaryVector-inl.h:50, Function:setInternalState, Expression: rawIndices_[i] < dictionaryValues_->size() (736 vs. 100) Dictionary index must be less than base vector's size. Index: 3., Source: RUNTIME, ErrorCode: INVALID_STATE
terminate called after throwing an instance of 'facebook::velox::VeloxRuntimeError'
  what():  Exception: VeloxRuntimeError
Error Source: RUNTIME
Error Code: INVALID_STATE
Reason: (736 vs. 100) Dictionary index must be less than base vector's size. Index: 3.
Retriable: False
Expression: rawIndices_[i] < dictionaryValues_->size()
Context: from_utf8(c0, c1)
Top-Level Context: and(like(from_utf8(c0, c1), 1oWy]lZR2x~JkQt>0%3cp)vY39X}ZBx|2*3NC;jGGBo(e!,'bP5fKv939>x4{F/%AQ[A;E,qWiCJg(4xK[_[]%H:VARCHAR, ^!pgm&XsZxssF{58Fa`tz[Nj\7NpQBf(lQ}jZJ<be4UT`8s'R_\{bQIRk$K,S1oE)qG)oShArW]P6stH&9!:|(U!:VARCHAR), like(from_utf8(c0, c1), 1oWy]lZR2x~JkQt>0%3cp)vY39X}ZBx|2*3NC;jGGBo(e!,'bP5fKv939>x4{F/%AQ[A;E,qWiCJg(4xK[_[]%H:VARCHAR, ^!pgm&XsZxssF{58Fa`tz[Nj\7NpQBf(lQ}jZJ<be4UT`8s'R_\{bQIRk$K,S1oE)qG)oShArW]P6stH&9!:|(U!:VARCHAR))
Function: setInternalState
File: buck-out/dev/gen/aab7ed39/velox/vector/velox_vector#header-mode-symlink-tree-with-header-map,headers/velox/vector/DictionaryVector-inl.h
Line: 50
Stack trace:
# 0  std::shared_ptr<facebook::velox::VeloxException::State const> facebook::velox::VeloxException::State::make<facebook::velox::VeloxException::make(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)::$_0>(facebook::velox::VeloxException::Type, facebook::velox::VeloxException::make(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)::$_0)
# 1  facebook::velox::VeloxException::VeloxException(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)
# 2  facebook::velox::VeloxRuntimeError::VeloxRuntimeError(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, std::basic_string_view<char, std::char_traits<char> >)
# 3  void facebook::velox::detail::veloxCheckFail<facebook::velox::VeloxRuntimeError, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>(facebook::velox::detail::VeloxCheckFailArgs const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
# 4  facebook::velox::DictionaryVector<facebook::velox::StringView>::setInternalState()
# 5  facebook::velox::DictionaryVector<facebook::velox::StringView>::DictionaryVector(facebook::velox::memory::MemoryPool*, boost::intrusive_ptr<facebook::velox::Buffer>, unsigned long, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer>, facebook::velox::SimpleVectorStats<facebook::velox::StringView> const&, std::optional<int>, std::optional<int>, std::optional<bool>, std::optional<int>, std::optional<int>)
# 6  decltype (::new ((void*)(0)) facebook::velox::DictionaryVector<facebook::velox::StringView>(std::declval<facebook::velox::memory::MemoryPool*&>(), std::declval<boost::intrusive_ptr<facebook::velox::Buffer>&>(), std::declval<unsigned long&>(), std::declval<std::shared_ptr<facebook::velox::BaseVector> >(), std::declval<boost::intrusive_ptr<facebook::velox::Buffer> >())) std::construct_at<facebook::velox::DictionaryVector<facebook::velox::StringView>, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(facebook::velox::DictionaryVector<facebook::velox::StringView>*, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 7  void std::allocator_traits<std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> > >::construct<facebook::velox::DictionaryVector<facebook::velox::StringView>, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >&, facebook::velox::DictionaryVector<facebook::velox::StringView>*, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 8  std::_Sp_counted_ptr_inplace<facebook::velox::DictionaryVector<facebook::velox::StringView>, std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >, (__gnu_cxx::_Lock_policy)2>::_Sp_counted_ptr_inplace<facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 9  std::__shared_count<(__gnu_cxx::_Lock_policy)2>::__shared_count<facebook::velox::DictionaryVector<facebook::velox::StringView>, std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(facebook::velox::DictionaryVector<facebook::velox::StringView>*&, std::_Sp_alloc_shared_tag<std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> > >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 10 std::__shared_ptr<facebook::velox::DictionaryVector<facebook::velox::StringView>, (__gnu_cxx::_Lock_policy)2>::__shared_ptr<std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(std::_Sp_alloc_shared_tag<std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> > >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 11 std::shared_ptr<facebook::velox::DictionaryVector<facebook::velox::StringView> >::shared_ptr<std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(std::_Sp_alloc_shared_tag<std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> > >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 12 std::shared_ptr<facebook::velox::DictionaryVector<facebook::velox::StringView> > std::allocate_shared<facebook::velox::DictionaryVector<facebook::velox::StringView>, std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> >, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(std::allocator<facebook::velox::DictionaryVector<facebook::velox::StringView> > const&, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 13 std::shared_ptr<facebook::velox::DictionaryVector<facebook::velox::StringView> > std::make_shared<facebook::velox::DictionaryVector<facebook::velox::StringView>, facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>, boost::intrusive_ptr<facebook::velox::Buffer> >(facebook::velox::memory::MemoryPool*&, boost::intrusive_ptr<facebook::velox::Buffer>&, unsigned long&, std::shared_ptr<facebook::velox::BaseVector>&&, boost::intrusive_ptr<facebook::velox::Buffer>&&)
# 14 std::shared_ptr<facebook::velox::BaseVector> facebook::velox::addDictionary<(facebook::velox::TypeKind)7>(boost::intrusive_ptr<facebook::velox::Buffer>, boost::intrusive_ptr<facebook::velox::Buffer>, unsigned long, std::shared_ptr<facebook::velox::BaseVector>)
# 15 facebook::velox::BaseVector::wrapInDictionary(boost::intrusive_ptr<facebook::velox::Buffer>, boost::intrusive_ptr<facebook::velox::Buffer>, int, std::shared_ptr<facebook::velox::BaseVector>)::$_0::operator()() const::{lambda()#1}::operator()() const
# 16 facebook::velox::BaseVector::wrapInDictionary(boost::intrusive_ptr<facebook::velox::Buffer>, boost::intrusive_ptr<facebook::velox::Buffer>, int, std::shared_ptr<facebook::velox::BaseVector>)::$_0::operator()() const
# 17 facebook::velox::BaseVector::wrapInDictionary(boost::intrusive_ptr<facebook::velox::Buffer>, boost::intrusive_ptr<facebook::velox::Buffer>, int, std::shared_ptr<facebook::velox::BaseVector>)
# 18 facebook::velox::DecodedVector::wrap(std::shared_ptr<facebook::velox::BaseVector>, facebook::velox::BaseVector const&, int)
# 19 facebook::velox::functions::(anonymous namespace)::FromUtf8Function::toVarcharNoCopy(std::shared_ptr<facebook::velox::BaseVector> const&, facebook::velox::DecodedVector&, facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx const&, std::shared_ptr<facebook::velox::BaseVector>&) const
# 20 facebook::velox::functions::(anonymous namespace)::FromUtf8Function::apply(facebook::velox::SelectivityVector const&, std::vector<std::shared_ptr<facebook::velox::BaseVector>, std::allocator<std::shared_ptr<facebook::velox::BaseVector> > >&, std::shared_ptr<facebook::velox::Type const> const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&) const
# 21 facebook::velox::exec::Expr::applyFunction(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 22 facebook::velox::exec::Expr::evalAllImpl(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 23 facebook::velox::exec::Expr::evalAll(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 24 facebook::velox::exec::Expr::evalWithNulls(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 25 facebook::velox::exec::Expr::evalEncodings(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 26 facebook::velox::exec::Expr::eval(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&, bool)
# 27 facebook::velox::exec::Expr::evalAllImpl(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 28 facebook::velox::exec::Expr::evalAll(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)::$_6::operator()(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&) const
# 29 void facebook::velox::exec::Expr::evaluateSharedSubexpr<facebook::velox::exec::Expr::evalAll(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)::$_6>(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&, facebook::velox::exec::Expr::evalAll(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)::$_6)
# 30 facebook::velox::exec::Expr::evalAll(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 31 facebook::velox::exec::Expr::evalWithNulls(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 32 facebook::velox::exec::Expr::evalEncodings(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 33 facebook::velox::exec::Expr::eval(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&, bool)
# 34 facebook::velox::exec::ConjunctExpr::evalSpecialForm(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 35 facebook::velox::exec::Expr::evalSpecialFormWithStats(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 36 facebook::velox::exec::Expr::evalAllImpl(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 37 facebook::velox::exec::Expr::evalAll(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 38 facebook::velox::exec::Expr::evalWithNulls(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 39 facebook::velox::exec::Expr::evalEncodings(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&)
# 40 facebook::velox::exec::Expr::eval(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&, bool)
# 41 facebook::velox::exec::ExprSet::eval(int, int, bool, facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::vector<std::shared_ptr<facebook::velox::BaseVector>, std::allocator<std::shared_ptr<facebook::velox::BaseVector> > >&)
# 42 facebook::velox::exec::ExprSet::eval(facebook::velox::SelectivityVector const&, facebook::velox::exec::EvalCtx&, std::vector<std::shared_ptr<facebook::velox::BaseVector>, std::allocator<std::shared_ptr<facebook::velox::BaseVector> > >&)
# 43 facebook::velox::test::ExpressionVerifier::verify(std::shared_ptr<facebook::velox::core::ITypedExpr const> const&, std::shared_ptr<facebook::velox::RowVector> const&, std::shared_ptr<facebook::velox::BaseVector>&&, bool, std::vector<unsigned int, std::allocator<unsigned int> >)
# 44 facebook::velox::test::ExpressionFuzzer::go()
# 45 facebook::velox::test::expressionFuzzer(std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<facebook::velox::exec::FunctionSignature const*, std::allocator<facebook::velox::exec::FunctionSignature const*> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::vector<facebook::velox::exec::FunctionSignature const*, std::allocator<facebook::velox::exec::FunctionSignature const*> > > > >, unsigned long)
# 46 FuzzerRunner::run(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, std::unordered_set<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
# 47 main
# 48 __libc_start_call_main
# 49 __libc_start_main_alias_2
# 50 _start

*** Aborted at 1680555623 (Unix time, try 'date -d @1680555623') ***
*** Signal 6 (SIGABRT) (0x37b3c000a5bfc) received by PID 678908 (pthread TID 0x7ff74efd7f40) (linux TID 678908) (maybe from PID 678908, UID 228156) (code: -6), stack trace: ***
    @ 000000000001100e folly::symbolizer::(anonymous namespace)::innerSignalHandler(int, siginfo_t*, void*)
                       ./folly/experimental/symbolizer/SignalHandler.cpp:449
    @ 000000000000f731 folly::symbolizer::(anonymous namespace)::signalHandler(int, siginfo_t*, void*)
                       ./folly/experimental/symbolizer/SignalHandler.cpp:470
    @ 000000000004459f (unknown)
    @ 000000000009c9d3 __GI___pthread_kill
    @ 00000000000444ec __GI_raise
    @ 000000000002c432 __GI_abort
    @ 00000000000a3fd4 __gnu_cxx::__verbose_terminate_handler()
    @ 00000000000a1b39 __cxxabiv1::__terminate(void (*)())
    @ 00000000000a1ba4 std::terminate()
    @ 00000000000a1ec1 __cxa_rethrow
    @ 00000000000261ec facebook::velox::test::ExpressionVerifier::verify(std::shared_ptr<facebook::velox::core::ITypedExpr const> const&, std::shared_ptr<facebook::velox::RowVector> const&, std::shared_ptr<facebook::velox::BaseVector>&&, bool, std::vector<unsigned int, std::allocator<unsigned int> >)
                       ./velox/expression/tests/ExpressionVerifier.cpp:159
    @ 00000000000b211c facebook::velox::test::ExpressionFuzzer::go()
                       ./velox/expression/tests/ExpressionFuzzer.cpp:1148
    @ 000000000010e226 facebook::velox::test::expressionFuzzer(std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<facebook::velox::exec::FunctionSignature const*, std::allocator<facebook::velox::exec::FunctionSignature const*> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::vector<facebook::velox::exec::FunctionSignature const*, std::allocator<facebook::velox::exec::FunctionSignature const*> > > > >, unsigned long)
                       ./velox/expression/tests/ExpressionFuzzer.cpp:1180
    @ 00000000002c454f FuzzerRunner::run(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, std::unordered_set<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
                       ./velox/expression/tests/FuzzerRunner.h:140
                       -> ./velox/expression/tests/ExpressionFuzzerTest.cpp
    @ 00000000002c3cf9 main
                       ./velox/expression/tests/ExpressionFuzzerTest.cpp:66
    @ 000000000002c656 __libc_start_call_main
    @ 000000000002c717 __libc_start_main_alias_2
    @ 00000000002b9990 _start
                       /home/engshare/third-party2/glibc/2.34/src/glibc-2.34/csu/../sysdeps/x86_64/start.S:116
Aborted (core dumped)
```

**Comment by bikramSingh91** [comment]
This looks similar to #4472, @kagamiori can you confirm and close one as duplicate (if required)

**Comment by kagamiori** [comment]
> @kagamiori if you can repro this internally, can you use, --persist_and_run_once to collect the input?

I don't think --persist_and_run_once would work with the failed test case I came across. The reason is that this test only failed when I ran fuzzer with the seed of the previous iteration.

**Comment by kagamiori** [comment]
> This looks similar to #4472, @kagamiori can you confirm and close one as duplicate (if required)

Confirmed that #4472 is the same. Closed #4472 as it's duplicated.
